src/Numerics/Mesh/Topologies.jl

```diff
@@ -171,6 +173,40 @@ struct BoxElementTopology{dim, T, nb} <: AbstractTopology{dim, T, nb}
     """
     bndytoface::NTuple{nb, Array{Int64, 1}}
 
+    """
+    Element to locally unique vertex number; `elemtouvert[v,e]` is the `v`th vertex 
+    of element `e`
+    """
+    elemtouvert::Union{Array{Int64, 2}, Nothing}
+
+    """
+    Vertex connectivity information for direct stiffness summation; 
+    `vtconn[lvt1, lelem1, lvt2, lelem2, ....]`
```

Comment:
```suggestion
    Vertex connectivity information for direct stiffness summation; 
    ```vtconn[vtconnoff[i]:vtconnoff[i+1]-1] == [lvt1, lelem1, lvt2, lelem2, ....]```
    where `lvt1` is the element-local vertex number of element `lelem1`, etc
```